### DISCUSSION TITLE
getting started in contributing

### DISCUSSION THREAD

**Discussion by AVI-SHUKLA1110** [protofire/solhint#502] [Q&A]
## I want to contribute

Greetings, all! I'm a web3 security enthusiast, and I've been diving into the world of Solidity lately. I'm eager to participate and make contributions to this repository to enhance my practical Solidity expertise and actively contribute to the projects. Are there any guidelines or helpful pointers to help me get started?

**Comment by dbale-arg** [comment]
Sorry the late response
I honestly did not check this section
There are no guidelines for contributing
Maybe open an issue, suggest a new rule or addition and see the feedback you receive back
You can check closed issues to see which ones were tagged as first issues and learn from there
Thanks a lot for posting, and sorry the late response
